HW7/HW_7_Task_3/HW_7_Task_3/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Linq;
+
+namespace HW_7_Task_3
+{
+    class Program
+    {
+        static void Main(string[] args)
```

Comment:
we will talk about this one. Just think how you can resolve this task on a paper